app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str) -> None:
+    cmd, src_file, dst_file = command.split(" ")
+
+    if cmd != "cp" or src_file == dst_file:
+        return
+
+    with open(src_file, "r") as file_in, open(dst_file, "w") as file_out:
+        file_out.write(file_in.read())
```

Comment:
This code will load the entire file into memory, which could be a problem if the file is very large. Consider using a loop to read the file in chunks and write each chunk to the destination file. This way, you can copy files of any size without using too much memory.